torchvision/transforms/transforms.py

```diff
@@ -199,7 +199,9 @@ class CenterCrop(object):
     """
```

Comment:
Doc above needs to be updated. And what about `(float, float)`?

Reply:
Just found the duplicated function and have made the changes. As for the (float, float) case, I didn't include it initially as I didn't think there would be a use for it, but greater flexibility is probably a good idea. I have made the changes. Tested with `(float, float)` input using `(0.875, 0.875)` and got 77.61% validation accuracy again. Waiting to see if the CI checks pass again, but there's a possible mutual import error from importing transforms (can you see if there is one?).